### PR TITLE
Adding image to cross namespace blog

### PR DESCRIPTION
**Pull request by yijie-04**
"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

- The graphic is added to the cross namespace event links blog

### CODE REVIEW

**Comment by Leo6Leo** [comment]
The image is not showing in the deploy-review

https://deploy-preview-6075--knative.netlify.app/blog/articles/cross-namespace-event-links-feature/

**Comment by yijie-04** [comment]
Hmm I'm not sure why it's not showing, is there an issue with the image?

File: blog/docs/articles/cross-namespace-event-links-feature.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 **Author: [Yijie Wang](https://www.linkedin.com/in/yijie-wang0806/){:target="_blank"}, LFX'24 Mentee**
 
+![Cross Namespace Event Links](/blog/docs/articles/images/cross-namespace-event-links.png)
```
Comment (Leo6Leo) [inline blog/docs/articles/cross-namespace-event-links-feature.md:5]:
```suggestion
![Cross Namespace Event Links](./images/cross-namespace-event-links.png)
```

**Pull request merged**